package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "iamdee",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "description": "Small AMD loader for inlining.",
-  "main": "dist/index.js",
+  "main": "dist/index.min.js",
   "scripts": {
-    "prepublish": "npm run lint && npm run test-original && npm run test && npm run clean && npm run build && npm run test-build",
+    "prepublishOnly": "npm run lint && npm run build && npm run test",
     "clean": "rm -rf ./dist/",
-    "build": "mkdir -p dist && ccjs src/index.js --compilation_level=ADVANCED_OPTIMIZATIONS | uglifyjs --mangle > dist/index.js",
-    "test": "karma start --singleRun=true",
-    "test-original": "REQUIRE_LIB=node_modules/requirejs/require.js karma start --singleRun=true",
-    "test-build": "REQUIRE_LIB=dist/index.js karma start --singleRun=true",
-    "tdd": "karma start",
-    "lint": "eslint src/ test/"
+    "ci": "npm run lint && npm run build && npm test",
+    "build": "tsc && npm run min-old && npm run min-modern",
+    "min": "npx google-closure-compiler --js=./dist/index.js --language_in=ECMASCRIPT5 --compilation_level=ADVANCED --language_out=ECMASCRIPT5 --define=IAMDEE_PRODUCTION_BUILD",
+    "min-old": "npm run min -- --js_output_file=dist/index.min.js",
+    "min-modern": "npm run min -- --js_output_file=dist/index.modern.min.js --define=IAMDEE_MODERN_BROWSER",
```

Comment:
should output ES6/ES2015

Reply:
When I enable that I get larger size after gzip...